docker/README.adoc

```diff
@@ -1,5 +1,8 @@
-==== docker build
-`docker build --force-rm=true --no-cache=true -t rhq-metrics:latest .`
```

Comment:
Right.  I'll update the Readme with instructions to launch Cassandra container and link with rhq-metrics the plain Docker way.